src/main/java/ai/promoted/delivery/client/TwoArmExperiment.java

```diff
@@ -0,0 +1,151 @@
+package ai.promoted.delivery.client;
+
+import ai.promoted.delivery.model.CohortArm;
+import ai.promoted.delivery.model.CohortMembership;
+
+/**
+ * Represents a two arm Experiment configuration.
+ *
+ * WARNING - while ramping up an experiment, do not change
+ * numControlBuckets or numTreatmentBuckets.  This will
+ * likely produce bad results.
+ *
+ * For the buckets, the treatment buckets are after the
+ * control buckets.
+ */
+public class TwoArmExperiment {
+  /**
+   * Name of cohort.
+   */
+  private final String cohortId;
+  
+  private final int cohortIdHash;
+  
+  /**
+   * Number of the numControlBuckets that are active.
+   */
+  private final int numActiveControlBuckets;
+  
+  /**
+   * Number of control buckets.
+   */
+  private final int numControlBuckets;
+  
+  /**
+   * Number of the numTreatmentBuckets that are active.
+   */
+  private final int numActiveTreatmentBuckets;
+  
+  /**
+   * Number of treatment buckets.
+   */
+  private final int numTreatmentBuckets;
+  
+  private final int numTotalBuckets;
+
+
+  /**
+   * Factory method for a 50/50 experiment.
+   * @param cohortId name of the cohort id (experiment)
+   * @param controlPercent percent to activate into control, range [0, 100]
+   * @param treatmentPercent percent to activate into treatment, range [0, 100]
+   * @return the configured experiment
+   */
+  public static TwoArmExperiment create5050TwoArmExperimentConfig(String cohortId, int controlPercent,
+      int treatmentPercent) {
+    if (controlPercent < 0 || controlPercent > 100) {
+      throw new IllegalArgumentException("Control percent must be in the range [0, 100]");
+    }
+    if (treatmentPercent < 0 || treatmentPercent > 100) {
+      throw new IllegalArgumentException("Treatment percent must be in the range [0, 100]");
+    }
+    return new TwoArmExperiment(cohortId, controlPercent, 100, treatmentPercent, 100);
```

Comment:
I'm okay with this.  I originally was thinking that we'd change all of the 100s here to 50s.

Reply:
I did that.